src/public/CowSwapGoerli.json

```diff
@@ -15,55 +20,55 @@
       "symbol": "GNO",
       "decimals": 18,
       "address": "0x02abbdbaaa7b1bb64b5c878f7ac17f8dda169532",
-      "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6810e776880C02933D47DB1b9fc05908e5386b96/logo.png"
+      "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6810e776880c02933d47db1b9fc05908e5386b96/logo.png"
     },
     {
       "name": "CoW Protocol Token",
       "chainId": 5,
       "symbol": "COW",
       "decimals": 18,
       "address": "0x91056D4A53E1faa1A84306D4deAEc71085394bC8",
-      "logoURI": "https://gnosis.mypinata.cloud/ipfs/Qme9B6jRpGtZsRFcPjHvA5T4ugFuL4c3SzWfxyMPa59AMo"
+      "logoURI": "https://gnosis.mypinata.cloud/ipfs/qme9b6jrpgtzsrfcpjhva5t4ugful4c3szwfxympa59amo"
```

Comment:
Another case that should not be lower cased 😅

Reply:
Thanks!
Fixed them all.
I actually did all fixes using scripts, so I reverted my changes, fixed script and ran the script again